runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/runtime/Platform.java

```diff
@@ -922,7 +922,7 @@ public static long getStateStamp() {
 	 * @since 3.0
 	 */
 	public static ILog getLog(Bundle bundle) {
```

Comment:
I think a comment mentioning the new method would be good here and below.
Eventually this method should at least be deprecated.